src/main/java/org/dependencytrack/resources/v1/ProjectResource.java

```diff
@@ -296,7 +309,12 @@ public Response updateProject(Project jsonProject) {
                     if (name == null) {
                         jsonProject.setName(project.getName());
                     }
-                    project = qm.updateProject(jsonProject, true);
+                    try {
+                        project = qm.updateProject(jsonProject, true);
+                    } catch (IllegalArgumentException e){
+                        LOGGER.error(e.getMessage());
```

Comment:
As this is a user error that administrators can't do anything about, it should not be logged as error. At the very least it should be `warn`, but I think really this should be `debug`. The response already tells the user what's wrong.